components/grid/grid.css

```diff
@@ -0,0 +1,33 @@
+:root{
+    --black:black;
+    --brown:brown:brown;
```

Comment:
I believe this is by mistake, you can fix this.

Reply:
yes, i have fixed it,thnks for the review